src/components/DaoCreator/index.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from "react";
+import { connect } from "react-redux";
+import { RouteComponentProps } from "react-router-dom";
+import DAOcreator from "@dorgtech/daocreator-ui-v1";
+import { enableWalletProvider, getWeb3Provider } from "arc";
+import { IRootState } from "reducers";
+import { showNotification } from "reducers/notifications";
+
+type IExternalProps = RouteComponentProps<any>;
+
+interface IDispatchProps {
+  showNotification: typeof showNotification;
+}
+
+type IProps = IDispatchProps;
+
+const mapStateToProps = (state: IRootState, ownProps: IExternalProps): IExternalProps => {
```

Comment:
we don't need mapStateToProps if its not actually importing any new props. you can just pass null to the connect call